ga4gh/frontend.py

```diff
@@ -261,6 +290,16 @@ def assertCorrectVersion(version):
         raise exceptions.VersionNotSupportedException()
 
 
+def authenticated(handler):
+    def checkAuthentication(*args, **kwargs):
+        requestToken = request.headers.get('Authorization')
+        requestToken = requestToken[7:] if requestToken else None
```

Comment:
we shouldn't have constants above 2 floating around in the code ... make it some kind of well-named variable so it's easy to see why we're using it here